### ISSUE TITLE
Compile to ES5

### ISSUE THREAD

**Issue by will-stone** [capaj/mobx-location#2]
Create-React-App yields the following error on build:

```
Failed to minify the code from this file:

        ./node_modules/mobx-location/src/mobx-location.js:17

Read more here: https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#npm-run-build-fails-to-minify
```

**Comment by will-stone** [comment]
PS, thanks for this little package, it's very useful when creating one's own MobX-powered router.

**Comment by capaj** [comment]
I will test it myself with CRA. I am using
```
  "module": "src/mobx-location.js",
```
in package.json. It should work fine. That file is ES6 compliant.

**Comment by will-stone** [comment]
It needs to be ES**5** compliant (according to the CRA docs). I notice you're now compiling to dist, so changing to

    "module": "dist/mobx-location.js",

should do the trick.

However, as you're using Parcel, this will always hash the name, making it tricky to use for plugins/libraries. https://github.com/parcel-bundler/parcel/issues/433

**Comment by capaj** [comment]
released in 0.3.1

**Comment by will-stone** [comment]
Nice. Thanks!
